omod/src/main/webapp/fragments/systemadmin/accounts/personDetails.gsp

```diff
@@ -97,6 +114,20 @@
                     ${ui.message("adminui.field.required")}
                 </span>
             </span>
+
+            <% customPersonFragments.each { sect ->
+                if(sect.extensionParams.type == 'personAttribute') {%>
+                ${ ui.includeFragment(sect.extensionParams.provider, sect.extensionParams.fragment, [
+                        formFieldName : sect.extensionParams.formFieldName,
+                        id : sect.extensionParams.formFieldName,
+                        title : ui.message(sect.extensionParams.title),
+                        label : ui.message(sect.extensionParams.label),
+                        initialValue : ui.message(account.getPersonAttribute(sect.extensionParams.uuid) != null ?
+                                account.getPersonAttribute(sect.extensionParams.uuid).value : '')
```

Comment:
Can we indent the above line correctly?

Reply:
That is the continue part of the above line. Does it contain wrong indent?